### PR TITLE
markdown(meandeck): add budget list

### PR DESCRIPTION
**Pull request by angrybacon**
https://deploy-preview-100--ddftwiki.netlify.app/meandeck/basics/

### CODE REVIEW

File: markdown/chapters/meandeck/basics.md [hunk 1]
```diff
@@ -145,4 +145,9 @@ of Will available.
 If you want to discover a bit by yourself before diving into more detailed
 chapters, try a couple goldfish games with the following proven list.
 
-<deck path="meandeck.ubrg.txt" />
+<deck path="meandeck.ubr.txt" />
+
+Alternatively, if you're looking into getting the cards here is a functional
+budget version:
+
+<deck path="meandeck.budget.txt" />
```
Comment (emidln) [inline markdown/chapters/meandeck/basics.md:153]: Obvious upgrades as budget allows:

#### Access to {{Force of Will}}:

```
-2 Flusterstorm
-2 Thoughtseize
+4 Force of Will
```

#### Access to 1+ {{Undergound Sea}}:

```
-1 Watery Grave
+1 Underground Sea
```

At 2 {{Underground Sea}}, {{Daze}} is unlocked as a reliable option.

**Review by emidln — APPROVED** [review]
I don't know if my comments make sense in the scope of the basics.md article. I'd approve as-is.

**Comment by Doishy** [comment]
Maybe not include in basics, maybe add list to faq as part of 15?

File: markdown/chapters/meandeck/basics.md (hunk 1 above)
Comment (angrybacon) [reply markdown/chapters/meandeck/basics.md:153]: Added in the 15th FAQ :pray:

**Pull request merged**